packages/flutter_tools/lib/src/commands/make_host_app_editable.dart

```diff
@@ -23,43 +21,18 @@ class MakeHostAppEditableCommand extends FlutterCommand {
     );
   }
 
-  FlutterProject _project;
-
   @override
   final String name = 'make-host-app-editable';
 
   @override
-  final String description = 'Moves host apps from generated directories to non-generated directories so that they can be edited by developers.\n\n'
-    'Use flags to specify which host app to make editable. If no flags are provided then all host apps will be made editable.\n\n'
-    'Once a host app is made editable, that host app cannot be regenerated by Flutter and it will not receive future template changes.';
+  bool get deprecated => true;
 
   @override
-  Future<void> validateCommand() async {
-    await super.validateCommand();
-    _project = FlutterProject.current();
-    if (!_project.isModule) {
-      throw ToolExit("Only projects created using 'flutter create -t module' can have their host apps made editable.");
-    }
-  }
+  final String description = 'Moves host apps from generated directories to non-generated directories so that they can be edited by developers.';
 
```

Comment:
Can we add a line here that says for modifications required in the host Android or iOS apps, make those modifications in your own wrapper apps that import the Flutter module?

Reply:
Good idea. I'll open a new PR.